File: Swift/L4/Lista_04_EX01/UsuarioConvidado.swift
public class UsuarioConvidado : Usuario{
    
    Override func int menu(){
        var opcao = 0
        
        println("----------MENU---------");
        println("(1)listarAutores");
        println("(2)contarCiencia");
        println("(3)listarFilosofia");
        println("(4)listarTudo");
        println("");
        printf("Digite uma opcao:");
        
        if(opcao<1 || opcao>7){
            opcao=0
        }        
        return opcao
    }
}

File: Java/Lista 4/L4_Ex1.1/src/l4_ex1/pkg1/UsuarioConvidado.java

package l4_ex1.pkg1;

import java.util.Scanner;

public class UsuarioConvidado extends Usuario{
    
    public UsuarioConvidado(String login) {
        super(login);
    }
    
    @Override
    public int menu(){
        int opcao = 0;
        Scanner ler = new Scanner( System.in );
        System.out.println("----------MENU---------");
        System.out.println("(1)listarAutores");
        System.out.println("(2)contarCiencia");
        System.out.println("(3)listarFilosofia");
        System.out.println("(4)listarTudo");
        
        System.out.println("");
        System.out.printf("Digite uma opcao:");
        opcao=ler.nextInt();
        
        if(opcao<1 || opcao>4){
            opcao=0;
        }        
        return opcao;
    }
    
}
